data_diff/__main__.py

```diff
@@ -200,24 +243,74 @@ def write_usage(self, prog: str, args: str = "", prefix: Optional[str] = None) -
     help="Name of run-configuration to run. If used, CLI arguments for database and table must be omitted.",
     metavar="NAME",
 )
+@click.option(
+    "--dbt",
+    is_flag=True,
+    help="Run a diff using your local dbt project. Expects to be run from a dbt project folder by default.",
+)
+@click.option(
+    "--dbt-cloud",
+    is_flag=True,
+    help="Run a diff using your local dbt project on Datafold cloud. Expects an api key on env var DATAFOLD_API_KEY.",
+)
```

Comment:
Wouldn't it be better to run this mode as `data-diff --dbt --cloud` — since there is already the `--cloud` option with this exact meaning?

Reply:
Agreed.

Reply:
altered so that the dbt functionality works like:
`data-diff --dbt` = local dbt diff
`data-diff --dbt --cloud` = cloud dbt diff

Reply:
Hm. I thought we had the `--cloud` option to run diffs in the cloud. Maybe I imagined that instead, but that mode was in the discussion for some time, so let's go that way by including the option for dbt scenario only and extending its meaning later.